src/Transition/CreateNewIDFUsingRulesV9_2_0.f90

```diff
@@ -480,6 +485,28 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               ! If your original object starts with W, insert the rules here
 
               ! If your original object starts with Z, insert the rules here
+              CASE('ZONEHVAC:EQUIPMENTLIST')
+                nodiff = .false.
```

Comment:
My FORTRAN is a bit rusty, but in my testing this worked. You might want to verify I'm doing all this right.